FreeBSD and OpenBSD define NULL as nullptr for C++11 or later.
https://github.com/freebsd/freebsd/commit/c8ed04c26b6758354853a6bed4629f71d0d01a7d
https://github.com/openbsd/src/commit/6ecde746dea9a5d17abf3bafa06c232b9189b33b

Source/GmmLib/TranslationTable/GmmPageTableMgr.cpp:415:21: error: non-pointer operand type 'GMM_GFX_ADDRESS' (aka 'unsigned long') incompatible with nullptr
    return AuxTTObj ? AuxTTObj->GetL3Address() : NULL;
                    ^ ~~~~~~~~~~~~~~~~~~~~~~~~
Source/GmmLib/TranslationTable/GmmPageTableMgr.cpp:474:28: error: invalid operands to binary expression ('GMM_GFX_ADDRESS' (aka 'unsigned long') and 'nullptr_t')
    if(GetAuxL3TableAddr() == NULL)
       ~~~~~~~~~~~~~~~~~~~ ^  ~~~~

--- Source/GmmLib/TranslationTable/GmmPageTableMgr.cpp.orig	2019-09-16 10:30:27 UTC
+++ Source/GmmLib/TranslationTable/GmmPageTableMgr.cpp
@@ -412,7 +412,7 @@ ERROR_CASE:
 /////////////////////////////////////////////////////////////////////////////////////
 GMM_GFX_ADDRESS GmmLib::GmmPageTableMgr::GetAuxL3TableAddr()
 {
-    return AuxTTObj ? AuxTTObj->GetL3Address() : NULL;
+    return AuxTTObj ? AuxTTObj->GetL3Address() : 0;
 }
 
 /////////////////////////////////////////////////////////////////////////////////////
@@ -471,7 +471,7 @@ GMM_STATUS GmmLib::GmmPageTableMgr::InitContextAuxTabl
 /////////////////////////////////////////////////////////////////////////////////////
 GMM_STATUS GmmLib::GmmPageTableMgr::UpdateAuxTable(const GMM_DDI_UPDATEAUXTABLE *UpdateReq)
 {
-    if(GetAuxL3TableAddr() == NULL)
+    if(GetAuxL3TableAddr() == 0)
     {
         GMM_ASSERTDPF(0, "Invalid AuxTable update request, AuxTable is not initialized");
         return GMM_INVALIDPARAM;
